tools/taints.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/DataDog/compute-go/table"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/planetlabs/draino/internal/cli"
+	"github.com/planetlabs/draino/internal/kubernetes"
+	"github.com/planetlabs/draino/internal/kubernetes/k8sclient"
+)
+
+type taintCommandFlags struct {
+	nodeName           string
+	nodegroupName      string
+	nodegroupNamespace string
+	outputFormat       cli.OutputFormatType
+	tableOutputParams  table.OutputParameters
+}
+
+var taintCmdFlags taintCommandFlags
+
+func TaintCmd(mgr manager.Manager) *cobra.Command {
+	taintCmd := &cobra.Command{
+		Use:        "taint",
+		SuggestFor: []string{"taint", "taints"},
+		Args:       cobra.MaximumNArgs(1),
```

Comment:
I think we could use minimum args here to make sure that it will be called with an argument.
```suggestion
		Args:       cobra.MinimumNArgs(1),
```

Reply:
done... I am always shared with these options, because we never know of subcommands are going to evolve. I am thinking that if we could set no constraint it would be better in fact.

Reply:
Tbh, it doesn't really matter. On the one hand, I don't think we will not have any problems without this config, and on the other hand, If we want to extend the tool, we can also change this number.